contracts/ark_starknet/src/interfaces.cairo

```diff
@@ -38,3 +38,9 @@ trait IExecutor<T> {
 trait IUpgradable<T> {
     fn upgrade(ref self: T, class_hash: ClassHash);
 }
+
+#[starknet::interface]
+trait IMaintenance<T> {
+    fn is_enabled(self: @T) -> bool;
+    fn enable(ref self: T, enable: bool);
+}
```

Comment:
could we use a more descriptive function name here ? like set_maintenance_mode

Reply:
Updated:
```cairo
#[starknet::interface]
trait IMaintenance<T> {
    fn is_in_maintenance(self: @T) -> bool;
    fn set_maintenance_mode(ref self: T, on: bool);
}
```